### ISSUE TITLE
[Suggestion] One-way glass

### ISSUE THREAD

**Issue by PopkaAkula** [tsunamayo/Starship-EVO#4836]
### Short Description of the suggestion:

So I noticed recently that material paints can paint windows an opaque color and I thought that it would be cool if this could be used for making one-way glass.

![image](https://user-images.githubusercontent.com/37738021/184617099-75f77fac-3b01-4238-b922-f6bb9b23538a.png)
[1] An image of a craft with gold painted windows

Of course, the glass becomes opaque on both sides because of how glass currently works. But a way one-way glass could be implemented is via stickers, by putting a sticker on one side and painting it with a material.

I don't know how feasible this is, just thought it would be a cool idea.

**Comment by tsunamayo** [comment]
@PopkaAkula Stickers dont work with transparent glass unfortunately. Nice looking ship, is it yours?

**Comment by PopkaAkula** [comment]
> @PopkaAkula Stickers dont work with transparent glass unfortunately. Nice looking ship, is it yours?

Yes it's on the workshop, though it doesn't normally have golden glass 🙃

**Comment by thriple** [comment]
Was just thinking of this. Would love some one-way mirrored glass as well.
